metagraph/integration_tests/test_api.py

```diff
@@ -39,7 +39,7 @@ def tearDownClass(cls):
         cls.server_process.kill()
 
     def _start_server(self, graph, annotation):
-        construct_command = '{exe} server_query -i {graph} -a {annot} --port {port} --address {host} -p {threads}'.format(
+        construct_command = '{exe} server_query -i {graph} -a {annot} --port {port} --address {host} -p {threads} --primary-kmers'.format(
```

Comment:
why do we need this flag?

Reply:
When in primary mode, setting the `--primary-kmers` flag makes the querying faster since it enables some optimizations in CanonicalDBG. If the input graph is canonical, then this flag is ignored.

Reply:
It's easy to make mistake if we always have to pass this. So let's turn this on by default for now and then add an attribute to primary graphs, so this can be inferred automatically.